examples/medplum-chart-demo/src/components/note-display/utils.ts

```diff
@@ -0,0 +1,52 @@
+import { Annotation, CodeableConcept, Coding, Quantity, QuestionnaireResponseItemAnswer } from '@medplum/fhirtypes';
+
+export interface Answers {
+  reasonForVisit: CodeableConcept;
+  date: string;
+  diastolic: Quantity;
+  systolic: Quantity;
+  height?: Quantity;
+  weight?: Quantity;
+}
+
+export function parseGenericAnswers(
```

Comment:
This is still too tightly coupled to the specific note types. Let's discuss together tomorrow how we can generalize the NoteDisplay logic